core/state/statedb.go

```diff
@@ -664,6 +686,10 @@ func (s *StateDB) createObject(addr common.Address) (newobj, prev *stateObject)
 	newobj = newObject(s, addr, nil)
 	if prev == nil {
 		s.journal.append(createObjectChange{account: &addr})
+		// TODO: add isPrecompile check
```

Comment:
@s1na I confirm indeed this is happening in my regression test suite, for now I'm checking via `evm.IsPrecompiledAddrr` to avoid emitting an even here.

Reply:
Added a way to check against list of active precompiles in statedb.